README.md

```diff
@@ -1 +1,61 @@
-# eoslime-shape-react
\ No newline at end of file
+# eoslime-shape-react
+
+The project is a complete and ready to use dApp with predefined React front-end and EOSLime integration. It consists of an EOS smart contract, deployment script, unit tests and a web part. The dApp represents a simple todo list, showing how to read and record a data on the blockchain through the browser. In gives a helpful example and foundation for a further development of more sophisticated dApp/s.
+
+In order to get the project up and running you need to do the following steps:
```

Comment:
Replace this with an Overview section and add

The shape represents a Todo Manager dApp showing you how to connect front-end to EOS blockchain 

[Add todomager image]

Functionality is as simple as

- Add a new Todo
- Move an existing Todo
- Remove an existing Todo

Reply:
updated